cpplint.py

```diff
@@ -5484,6 +5485,19 @@ def CheckLanguage(filename, clean_lines, linenum, file_extension,
           'https://google-styleguide.googlecode.com/svn/trunk/cppguide.xml#Namespaces'
           ' for more information.')
 
+  # Check namespace names for correct naming, according to
+  # https://google.github.io/styleguide/cppguide.html#Namespace_Names
+  # namespace names are "all lower-case, with words separated by underscores"
+  match = re.match(r'.*namespace\s+([^{=;]+)\s+[{;=]', line)
+  if match:
+    name = match.group(1)
+    if name != "" and not re.match(r'^((inline )?[a-z_]+(::)?)+$', name):
+      error(filename, linenum, 'build/namespaces_names', 4,
```

Comment:
```suggestion
      error(filename, linenum, 'naming/namespaces', 4,
```
This is decidedly not a building issue. This entire check is also not suited for the language rules−checking function. (I recommend you move this to a new function, which should also solve the lint errors.